GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/DehydrateTests.cs

```diff
@@ -176,279 +176,264 @@ public void DehydrateShouldFailOnWrongDiskLayoutVersion()
+            string folderToDelete = "GitCommandsTests";
+            string folderToDeleteVirtualPath = this.Enlistment.GetVirtualPathTo(folderToDelete);
+            this.fileSystem.DeleteDirectory(folderToDeleteVirtualPath);
             GitProcess.Invoke(this.Enlistment.RepoRoot, "reset --hard");
 
-            string folderToDehydrate = Path.Combine("GitCommandsTests", "DeleteFileTests");
+            string folderToDehydrate = Path.Combine(folderToDelete, "DeleteFileTests");
             this.Enlistment.GetVirtualPathTo(folderToDehydrate).ShouldBeADirectory(this.fileSystem);
 
-            this.DehydrateShouldSucceed(new[] { $"Cannot dehydrate folder '{folderToDehydrate}': Must dehydrate parent folder 'GitCommandsTests/'." }, confirm: true, noStatus: false, foldersToDehydrate: folderToDehydrate);
+            this.DehydrateShouldSucceed(new[] { $"Cannot dehydrate folder '{folderToDehydrate}': Must dehydrate parent folder '{folderToDelete}/'." }, confirm: true, noStatus: false, foldersToDehydrate: folderToDehydrate);
         }
 
-        [TestCase]
+        [TestCase]
         public void FolderDehydrateDirtyStatusShouldFail()
         {
-            string fileToCreate = this.Enlistment.GetVirtualPathTo(Path.Combine("GVFS", $"{nameof(this.FolderDehydrateDirtyStatusShouldFail)}.txt"));
-            this.fileSystem.WriteAllText(fileToCreate, "new file contents");
-            fileToCreate.ShouldBeAFile(this.fileSystem);
+            string folderToDehydrate = "GVFS";
+            string fileToCreateVirtualPath = this.Enlistment.GetVirtualPathTo(Path.Combine(folderToDehydrate, $"{nameof(this.FolderDehydrateDirtyStatusShouldFail)}.txt"));
+            this.fileSystem.WriteAllText(fileToCreateVirtualPath, "new file contents");
+            fileToCreateVirtualPath.ShouldBeAFile(this.fileSystem);
 
-            this.DehydrateShouldFail(new[] { "Running git status...Failed", "Untracked files:", "git status reported that you have dirty files" }, noStatus: false, foldersToDehydrate: "GVFS");
+            this.DehydrateShouldFail(new[] { "Running git status...Failed", "Untracked files:", "git status reported that you have dirty files" }, noStatus: false, foldersToDehydrate: folderToDehydrate);
             GitProcess.Invoke(this.Enlistment.RepoRoot, "clean -xdf");
         }
 
-        [TestCase]
+        [TestCase]
         public void FolderDehydrateDirtyStatusWithNoStatusShouldFail()
         {
-            string fileToCreate = this.Enlistment.GetVirtualPathTo(Path.Combine("GVFS", $"{nameof(this.FolderDehydrateDirtyStatusWithNoStatusShouldFail)}.txt"));
-            this.fileSystem.WriteAllText(fileToCreate, "new file contents");
-            fileToCreate.ShouldBeAFile(this.fileSystem);
+            string folderToDehydrate = "GVFS";
+            string fileToCreateVirtualPath = this.Enlistment.GetVirtualPathTo(Path.Combine(folderToDehydrate, $"{nameof(this.FolderDehydrateDirtyStatusWithNoStatusShouldFail)}.txt"));

```

Comment:
Similar minor comment here, you can drop the `Path.Combine` and use `GetVirtualPathTo(params string[] pathParts)`